pr-data.csv

```diff
@@ -1749,9 +1751,10 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors,org.apache.nifi.hbase.TestFetchHBaseRow.testFetchToAttributesWithStringValues,ID,Accepted,https://github.com/apache/nifi/pull/5758,the PR itself was rejected but the commit  https://github.com/apache/nifi/commit/22b47ccc7b8af2a432ac46e893388da1afefa611 was added
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors,org.apache.nifi.hbase.TestFetchHBaseRow.testFetchToContentWithQualifierAndValueJSON,ID,Accepted,https://github.com/apache/nifi/pull/5758,the PR itself was rejected but the commit  https://github.com/apache/nifi/commit/22b47ccc7b8af2a432ac46e893388da1afefa611 was added
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors,org.apache.nifi.hbase.TestFetchHBaseRow.testFetchToContentWithStringValues,ID,Accepted,https://github.com/apache/nifi/pull/5758,the PR itself was rejected but the commit  https://github.com/apache/nifi/commit/22b47ccc7b8af2a432ac46e893388da1afefa611 was added
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service,org.apache.nifi.hbase.TestHBase_1_1_2_ClientService.testScan,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service,org.apache.nifi.hbase.TestHBase_1_1_2_ClientService.testScan,ID,Accepted,https://github.com/apache/nifi/pull/6632,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-hbase_2-client-service-bundle/nifi-hbase_2-client-service,org.apache.nifi.hbase.TestHBase_2_ClientService.testScan,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors,org.apache.nifi.hbase.TestScanHBase.testScanToContentWithStringValues,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors,org.apache.nifi.hbase.TestScanHBase.testScanToContentWithStringValues,ID,Accepted,https://github.com/apache/nifi/pull/6633,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors,org.apache.nifi.hbase.TestScanHBase.testScanToContentWithQualifierAndValueJSON,ID,Accepted,https://github.com/apache/nifi/pull/6633,
```

Comment:
Did you find the new tests on the old SHA `2bd752d868a8f3e36113b078bb576cf054e945e8` (from Nov 2021), or did you find it on the latest SHA that you worked with?

Reply:
I ran the commands for `2bd752d868a8f3e36113b078bb576cf054e945e8` and the flaky test existed in this SHA. Is it okay if I keep the SHA value as it is then?